### DISCUSSION TITLE
False positive?

### DISCUSSION THREAD

**Discussion by akoppa** [SubtitleEdit/subtitleedit#5297] [Q&A]
In SubtitleEdit version 3.6.2 Malwarebytes reports:
Malware.AI.4225176342, C:\PROGRAM FILES\SUBTITLE EDIT\SUBTITLEEDIT.EXE, 

AdwCleaner, RogueKiller and Diag says it's clean.

**Comment by niksedk** [comment]
Please report "false positive" to your anti-virus provider - thx :)

Try to re-download/install in a few days + update your anti-virus - then everything is hopefully okay.

See https://github.com/SubtitleEdit/subtitleedit/issues/5280

PS: I've reported it too... but the more the merrier :)
https://forums.malwarebytes.com/topic/278185-false-positive/
